test/apkPath_test.js

```diff
@@ -1,10 +1,11 @@
 var assert = require('assert');
 var apkPath = require('../src/apkPath');
+var path = require('path');
 
 describe('apkPath', function() {
     before(function(){
         settings = {}
-        settings.download_path = process.cwd() + '\\test\\downloads';
+        settings.download_path = path.win32.join(process.cwd(),'\\test\\downloads');
```

Comment:
the whole point is you should never embed the path separator in the string but provide a list of folder name and join it as path.join